src/lib.rs

```diff
@@ -216,6 +242,65 @@ impl Default for CertificateParams {
 	}
 }
 
+impl CertificateParams {
+	/// Parses the ca certificate from the ASCII PEM format
+	/// 
+	/// See `from_ca_cert_der` for more details.
+	#[cfg(all(feature = "pem", feature = "x509-parser"))]
+	pub fn from_ca_cert_pem(pem_str :&str, key_pair: KeyPair) -> Result<Self, RcgenError> {
```

Comment:
style nit: please put the space before the colon.

Reply:
Done